modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/PlaybackManager.kt

```diff
@@ -1294,20 +1294,12 @@ open class PlaybackManager @Inject constructor(
 
         val durationDiffSeconds = (durationMs - episode.durationMs) / 1000
         if (abs(durationDiffSeconds) > 0) {
-            val notifyUser = abs(durationDiffSeconds) > 30
-            if (notifyUser) {
-                LogBuffer.e(LogBuffer.TAG_PLAYBACK, "The total episode duration has changed significantly ($durationDiffSeconds seconds)")
-                launch(Dispatchers.Main) {
-                    val message = application.getString(LR.string.episode_duration_change, durationDiffSeconds)
```

Comment:
Should we remove the string `episode_duration_change` too?

Reply:
Ah, good catch! Removed in https://github.com/Automattic/pocket-casts-android/pull/1353/commits/47dc0234486e9a9d1205877a777d0aa7f2f4e54a.